pkg/networkserver/downlink.go

```diff
@@ -676,6 +718,9 @@ func (ns *NetworkServer) processDownlinkTask(ctx context.Context) error {
 
 					default:
 						// Data downlink for Class B/C in Rx1
+						req.Rx1Delay = dev.MACState.CurrentParameters.Rx1Delay
+						req.Rx1Frequency = rx1Freq
```

Comment:
Why don't we also use `downlinkPathsForClassA` for Rx1 of a Class B/C device?

Reply:
Good call. Was planning on fixing that in #396 but will apply in this PR.

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/410/commits/783b87b66d6c3568e98243a9dfda21bf92cb541d